Release notes for v0.8.9 and v0.8.10

penrods opened the pull request.

Commit:
Release notes for v0.8.9 and v0.8.10

btotharye committed:
Updated docs.md (#95)

Fixed broken link for the hello world script.

bdice committed:
Updated links. (#94)

I updated the Skills links to point to the newer separate repositories.

bdice committed:
Fixed broken link to Skills page. (#93)

Fixed broken link to Skills page.

aatchison committed:
removed unneeded files; moved pages to root (#96)

aatchison committed:
removed hidden files

danielquinn committed:
fix: use the correct "you're"

kfezer committed:
Fixed Broken Link

As per https://github.com/MycroftAI/mycroft-core-documentation/issues/100

kfezer committed:
Fixing Broken link for Hello World SKill

as  noted in 
https://github.com/MycroftAI/mycroft-core-documentation/issues/100

kfezer committed:
Changed defualt location for skills

changed to show in /opt/

djmitche committed:
Update dev dependencies for Fedora 25

This is based on my experience installing the dependencies, and thus
includes a few things not present in the old F24 list.

Commit:
Merge pull request #101 from djmitche/fedora25

Update dev dependencies for Fedora 25

shifubear committed:
Fixed typos in documentation (#105)

kfezer committed:
Merge pull request #98 from danielquinn/patch-1

fix: use the correct "you're"

kfezer committed:
Fixing some broken links and typos
	modified:   06.skill.creation/01.creating-first-skill/docs.md
	modified:   06.skill.creation/02.vocabulary/docs.md
	modified:   06.skill.creation/04.skills-sdk/docs.md
	modified:   06.skill.creation/docs.md

kfezer committed:
Merge pull request #106 from MycroftAI/fixes-after-skill-abastraction

	Fixing some broken links and typos

aatchison committed:
Update README.md

aatchison committed:
Update README.md

kfezer committed:
modified:   02.installing.and.running/01.installation/docs.md
	renamed:    02.installing.and.running/03.running/docs.md -> 02.installing.and.running/02.running/docs.md
	renamed:    02.installing.and.running/02.pairing/docs.md -> 02.installing.and.running/03.pairing/docs.md
	new file:   02.installing.and.running/05.cheatsheet/docs.md
	modified:   02.installing.and.running/docs.md

kfezer committed:
modified:   02.installing.and.running/01.installation/docs.md
	renamed:    02.installing.and.running/03.running/docs.md -> 02.installing.and.running/02.running/docs.md
	renamed:    02.installing.and.running/02.pairing/docs.md -> 02.installing.and.running/03.pairing/docs.md
	new file:   02.installing.and.running/05.cheatsheet/docs.md
	modified:   02.installing.and.running/docs.md

kfezer committed:
Fixed some typos and name titles
	modified:   02.installing.and.running/03.pairing/docs.md
	modified:   02.installing.and.running/05.cheatsheet/docs.md
	modified:   03.skills.and.features/02.config/docs.md

kfezer committed:
Updates to config details and names
	modified:   02.installing.and.running/05.cheatsheet/docs.md
	modified:   03.skills.and.features/02.config/docs.md

kfezer committed:
Merge branch 'dev' into pycon-revisions-fix

kfezer committed:
Merge pull request #114 from MycroftAI/pycon-revisions-fix

Pycon revisions fix

kfezer committed:
Adding a few more terminal commands
	modified:   02.installing.and.running/02.running/docs.md

kfezer committed:
Merge pull request #115 from MycroftAI/more-running-instructions

	Adding a few more terminal commands

ajuhl committed:
Remove reference to load data files

btotharye committed:
adding in mark1 docs

btotharye committed:
adding in mark1 docs

btotharye committed:
removing extra release notes

btotharye committed:
fixing folder structure

btotharye committed:
updating files

btotharye committed:
updating for password section

btotharye committed:
adding in more mark 1 info

btotharye committed:
adding in more mark 1 info

btotharye committed:
Update docs.md

btotharye committed:
adding in command list

btotharye committed:
adding in command list

btotharye committed:
updated the user guide url

btotharye committed:
added in access menu

btotharye committed:
final updates on passwordless ssh access

btotharye committed:
Update docs.md

kfezer committed:
Re-order Stucture so Mark1 is higher
	renamed:    07.mark-1/01.setting-up/docs.md -> 02.mark-1/01.setting-up/docs.md
	renamed:    07.mark-1/02.passwordless.ssh/docs.md -> 02.mark-1/02.passwordless.ssh/docs.md
	renamed:    07.mark-1/03.using.mark1/docs.md -> 02.mark-1/03.using.mark1/docs.md
	renamed:    07.mark-1/docs.md -> 02.mark-1/docs.md
	renamed:    02.installing.and.running/01.installation/01.git.clone.install/docs.md -> 03.installing.and.running/01.installation/01.git.clone.install/docs.md
	renamed:    02.installing.and.running/01.installation/02.raspberry.pi/docs.md -> 03.installing.and.running/01.installation/02.raspberry.pi/docs.md
	renamed:    02.installing.and.running/01.installation/03.kde.desktop/docs.md -> 03.installing.and.running/01.installation/03.kde.desktop/docs.md
	renamed:    02.installing.and.running/01.installation/04.apt-get.install/docs.md -> 03.installing.and.running/01.installation/04.apt-get.install/docs.md
	renamed:    02.installing.and.running/01.installation/05.docker.install/docs.md -> 03.installing.and.running/01.installation/05.docker.install/docs.md
	renamed:    02.installing.and.running/01.installation/docs.md -> 03.installing.and.running/01.installation/docs.md
	renamed:    02.installing.and.running/02.running/docs.md -> 03.installing.and.running/02.running/docs.md
	renamed:    02.installing.and.running/03.pairing/docs.md -> 03.installing.and.running/03.pairing/docs.md
	renamed:    02.installing.and.running/04.FAQ/01.audio.issues/docs.md -> 03.installing.and.running/04.FAQ/01.audio.issues/docs.md
	renamed:    02.installing.and.running/04.FAQ/docs.md -> 03.installing.and.running/04.FAQ/docs.md
	renamed:    02.installing.and.running/05.cheatsheet/docs.md -> 03.installing.and.running/05.cheatsheet/docs.md
	renamed:    02.installing.and.running/docs.md -> 03.installing.and.running/docs.md
	renamed:    03.skills.and.features/01.adding.skills/docs.md -> 04.skills.and.features/01.adding.skills/docs.md
	renamed:    03.skills.and.features/02.config/docs.md -> 04.skills.and.features/02.config/docs.md
	renamed:    03.skills.and.features/docs.md -> 04.skills.and.features/docs.md
	renamed:    04.contributing/docs.md -> 05.contributing/docs.md
	renamed:    05.overview/01.messagebus/docs.md -> 06.core.overview/01.messagebus/docs.md
	renamed:    05.overview/docs.md -> 06.core.overview/docs.md
	renamed:    06.skill.creation/01.creating-first-skill/docs.md -> 07.skill.creation/01.creating-first-skill/docs.md
	renamed:    06.skill.creation/02.vocabulary/docs.md -> 07.skill.creation/02.vocabulary/docs.md

kfezer committed:
Merge pull request #116 from ajuhl/fixNewSkill

Remove reference to load data files

kfezer committed:
Merge branch 'feature/reorg-mark1' into dev

kfezer committed:
Merge branch 'dev' of https://github.com/MycroftAI/mycroft-core-documentation into dev

kfezer committed:
Adding Tutorial for GPIO interaction
	new file:   07.skill.creation/05.advanced-topics/03.gpio/docs.md

kfezer committed:
Merge pull request #121 from MycroftAI/feature/add-gpio-tutorial-section

	Adding Tutorial for GPIO interaction

kfezer commented:
This needs to get adapted to the new style. Next release update from these docs.

kfezer committed:
Fixed broken links, add a new regex example
	modified:   07.skill.creation/05.advanced-topics/01.regex/docs.md

kfezer committed:
Merge pull request #122 from MycroftAI/bug/link-corrections

	Fixed broken links, add a new regex example

kfezer committed:
More guides in regex section
	modified:   07.skill.creation/05.advanced-topics/01.regex/docs.md

kfezer committed:
Merge pull request #123 from MycroftAI/feature/more-regex

	More guides in regex section

kfezer reviewed (changes requested):
Redo this on the new structure for this repo

geekedoutsolutions committed:
Update docs.md

vavrek committed:
Merge pull request #124 from geekedoutsolutions/patch-1

Update docs.md

btotharye committed:
Updated docs for docker install

Updated the docs to include the information from the docker repo and goes over how to setup docker.

vavrek committed:
Merge pull request #125 from btotharye/patch-2

Updated docs for docker install

oren committed:
added ubuntu cheatsheet

oren committed:
add msm to cheatsheet

oren committed:
add 2 sections under 'add a skill' - manuall and using msm.

oren committed:
more commands for Mark 1

LearnedVector committed:
Fix/skill framework documentation (#127)

* Update docs.md

* Update docs.md

* Update docs.md

* Update docs.md

LearnedVector committed:
Merge pull request #126 from oren/dev

added ubuntu cheatsheet

LearnedVector committed:
Update docs.md

Commit:
Rename to fix onboarding email link

Previous update renamed section 3, breaking the target URL
used by the onboarding email:
https://docs.mycroft.ai/development

rencire committed:
Replaced section about Slack with link to Mattermost

LearnedVector committed:
Merge pull request #128 from rencire/rencire-patch-1

Replaced section about Slack with link to Mattermost

aatchison committed:
Updating to reflect removal of build host setup scripts
